src/changelog/2024/08/ldap-sso-groups.md

```diff
@@ -0,0 +1,46 @@
+---
+title: LDAP Single Sign On Updates
+description: "Team membership controlled by LDAP groups"
+
+date: 2024-08-28 13:00:00.0
+
+authors: ["ben-hardill"]
+tags:
+    - sso
+---
+
+In the v2.8.0 release we have updated the LDAP SSO feature to allow group
+membership to be managed by LDAP groups.
+
+## Configuring
+
+Building on the SAML based SSO configuration there is now a `Manage roles using group assertions` check box on the LDAP SSO configuration page. This allows a base LDAP DN for groups to be set. This will used to look up which groups a user is a member of when they login to the platform.
```

Comment:
`LDAP DN` means?

Reply:
> there is now a `Manage roles using group assertions` check box on the LDAP SSO configuration page.

Screenshot?

Reply:
Destinguished Name.

If you don't know you shouldn't be making these changes.